### ISSUE TITLE
We need a standup bot!

### ISSUE THREAD

**Issue by missaugustina** [BonnyCI/projman#150]
For specifics, see https://github.com/BonnyCI/quartermaster

**Comment by rattboi** [comment]
We were going to use bonnyci/quartermaster, but that's on hold. I'll put my investigations here as I find them.

**Comment by missaugustina** [comment]
The last comment indicates this would be updated with investigation results. Are there any updates?

**Comment by rattboi** [comment]
We've decided to leave the original quartermaster behind and move forward with a new bot based on errbot. There are a series of tasks involved in standing up the bot and starting to use it. This can be tracked here: https://github.com/BonnyCI/projman/issues/174

Since this issue is for the standup aspect of the bot, I'm going to use this for standup functionality status.

I've got a WIP PR for the bot extensions repo here: bonnyci/errbot-extensions/pull/3

I'll continue adding features next week.
